docs/administration/back_office/search_sorting.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Add a "sort by" method to the Back Office search result page.
+---
+
+# Customize search sorting
+
+To add an entry to the "Sort by" to the Back Office search result page, create a service to implement the `Ibexa\Contracts\Search\SortingDefinition\SortingDefintionProviderInterface` and tag `ibexa.search.sorting_definition.provider`.
```

Comment:
Some introduction is missing here, like: "You can customize search sorting by adding an entry (...). To do it, (...)".

Reply:
@julitafalcondusza What about this?:
```suggestion
You can customize the **Sort by** menu in the Back Office search result page. To do so, create a service implementing the `Ibexa\Contracts\Search\SortingDefinition\SortingDefintionProviderInterface` and tag it `ibexa.search.sorting_definition.provider`.
```